packages/scheduled-tasks/package.json

```diff
@@ -58,6 +58,9 @@
 		"sqs-consumer": "^5.6.0",
 		"sqs-producer": "^2.1.0"
 	},
+	"peerDependencies": {
+		"@sapphire/stopwatch": "^1.2"
+	},
```

Comment:
Remove this and add it to the README instead. You can peek at i18next on how to format it, it goes in the list where framework is also mentioned.

Peer dependencies are a PITA with npm. 

Doing this change will also fix CI.